wasm/jsapi/memory/grow.any.js

```diff
@@ -1,10 +1,9 @@
-// META: global=jsshell
+// META: global=default,jsshell
 
 function assert_ArrayBuffer(actual, expected, message) {
   // https://github.com/WebAssembly/spec/issues/840
-  const bufferType = expected.shared ? self.SharedArrayBuffer : ArrayBuffer;
-  assert_equals(Object.getPrototypeOf(actual), bufferType.prototype,
-                `${message}: prototype`);
```

Comment:
Can we add a check for `Object.getPrototypeOf(actual).toString()` or something like that?